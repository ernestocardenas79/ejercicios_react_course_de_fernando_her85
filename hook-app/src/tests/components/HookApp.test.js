import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import { HookApp } from "../../HookApp";

describe("Prueba de componente <GiftGrid />", () => {
  test("Debe de crearce el componente", () => {
    const wrapper = shallow(<HookApp />);

    expect(wrapper).toMatchSnapshot();
  });
});
