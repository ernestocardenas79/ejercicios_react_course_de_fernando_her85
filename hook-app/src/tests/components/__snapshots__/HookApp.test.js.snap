// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de componente <GiftGrid /> Debe de crearce el componente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
